File: C/beginner/1008.c
/*Including Libraries*/
#include <stdio.h>

/*Main Function*/
int main(){
    /*Variable declaration and initialization*/
    int doc, hours;
    float hour_value, total_income;

    /*Requesting values to user*/
    scanf("%d", &doc);
    scanf("%d", &hours);
    scanf("%f", &hour_value);

    /*Doing the calculations*/
    total_income = hours * hour_value;

    /*Showing results*/
    printf("NUMBER = %d\n", doc);
    printf("SALARY = U$ %.2f\n", total_income);

    /*Ending algorithm*/
    return 0;
}

/*By Fernando Serra*/
/*https://github.com/fernandosserra*/

File: Haskell/beginner/1008.hs
-- Importing files
import Text.Printf

-- Main Function
main = do
    -- Requesting values and assign into variables
    doc <- readLn :: IO Int
    hours <- readLn :: IO Int
    hour_value <- readLn :: IO Float
    
    -- Doing the calculations
    let total_income = fromIntegral hours * hour_value
    -- Nice to see, in Haskell we cannot multiply int and float directly
    
    -- Showing results
    printf "NUMBER = %d\n" doc
    printf "SALARY = U$ %.2f\n" total_income

-- By Fernando Serra
-- https://github.com/fernandosserra